external/dhcpc/dhcpc.c

```diff
@@ -659,10 +670,13 @@ static int dhcpc_request(void *handle, struct dhcpc_state *presult)
 		 (presult->netmask.s_addr >> 8) & 0xff,
 		 (presult->netmask.s_addr >> 16) & 0xff,
 		 (presult->netmask.s_addr >> 24) & 0xff);
-	ndbg("Got DNS server %d.%d.%d.%d\n", (presult->dnsaddr.s_addr) & 0xff,
-		 (presult->dnsaddr.s_addr >> 8) & 0xff,
-		 (presult->dnsaddr.s_addr >> 16) & 0xff,
-		 (presult->dnsaddr.s_addr >> 24) & 0xff);
+
+	for (itr = 0; itr < total_dns_servers; ++itr) {
+		ndbg("Got DNS server %d.%d.%d.%d\n", (presult->dnsaddr[itr].s_addr) & 0xff,
```

Comment:
in my opinion, could you add function for it?
i mean, 
```
print_func(const char * str, (type)s_addr) {
       ndbg("str %d.%d.%d.%d\n", s_addr & 0xff, ....)
}
```
Because the same part is reused on lines 665 and 670.

Reply:
there is no point to add a function for a single line. in addition, here we are using custom print statements like got DNS server, got netmask, got IP address, etc. so if we add a function for it then there will be a difficulty for these print statements.